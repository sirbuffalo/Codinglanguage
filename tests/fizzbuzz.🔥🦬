for i of [1...30]
    if i % 15 == 0
        print(-15)
    elif i % 3 == 0
        print(-3)
    elif i % 5 == 0
        print(-5)
    else
        print(i)