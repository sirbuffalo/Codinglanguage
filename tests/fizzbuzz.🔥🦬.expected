1
2
-3
3
4
-5
-3
6
7
8
-3
9
-5
11
-3
12
13
14
-15
15
16
17
-3
18
19
